core/domain/wipeout_service_test.py

```diff
@@ -275,7 +277,8 @@ def test_pre_delete_profile_users_works_correctly(self):
         self.assertTrue(user_settings.deleted)
 
         user_auth_details = (
-            auth_models.UserAuthDetailsModel.get_by_id(self.profile_user_id))
+            auth_models.UserAuthDetailsModel.get_by_id(
```

Comment:
Should this be ``get()`` now? Ditto elsewhere below where we're still using ``get_by_id``.

Reply:
https://github.com/oppia/oppia/pull/11914#discussion_r578965289

Reply:
Resolving this as this is being discussed in the above conversation